platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,482 @@
+
+###### Examples Scenarios:
+
+* Data derived from platform usage metrics inform decisions to allocate funds and staff to the most impactful initiatives.
+
+##### Level 4, Optimizing - Enabled ecosystem
+
+Platform teams find ways to increase organization-wide efficiency and effectiveness beyond basic capabilities. Core platform maintainers intentionally strive to optimize time-to-market for new products, reduce costs across the enterprise, enable efficient governance and compliance for new services, scale workloads quickly and easily, and other cross-cutting requirements. These core maintainers are focused on enabling capability specialists to seamlessly integrate their requirements and offerings into existing and new parts of platforms. Further, the organization focuses people and resources from specialist domains like security, performance, quality on engaging with provided platform frameworks to introduce advanced features that can enable product teams to accelerate their adherence to company goals without depending on a centralized team backlog.
+
+###### Characteristics:
+
+* It becomes a priority to enable specialists to extend platform capabilities and introduce new ones
+* The organization can centralize specialists allowing their knowledge and support to be spread through platform capabilities
+
+###### Examples Scenarios:
+
+* Marketing works with platform builders to introduce consistent user tracking in order to attribute marketing efforts to product outcomes. 
+* Automation initiative reduces human time to provision databases by 30 minutes per instance, saving $10m/year
+
+
+### Adoption
+<h4 style="color:gray"><i>Why and how do users discover and use internal platforms and platform capabilities?</i></h4>
+
+Adoption describes not only how and how much an organization uses platform capabilities, but also what motivates them to do so. In the early stages,  many target users may not realize they are using a platform at all, rather they see their tools as an ad hoc collection of capabilities from various internal sources. This may mature into a group of capabilities that is consistently managed and presented to users - that is, one or more platforms. As the capabilities become more refined and discoverable, it is common that the drive for platform use moves away from more external motivations like mandates or incentives. Allowing users to self-select into platform capabilities and ideally even invest their own efforts into the wider platform ecosystem.
+
+
+<p align="center">
+<img src="assets/adoption-curve.jpg" width=600px />
+<br/>
+<em>A diagram to indicate a common growth pattern for platform</br>adoption. This showcases the often slow start driven mainly by platform builders.</br></br>Once platforms provides enough value to users, growth becomes more pulled by</br>the users causing a steeper adoption curve.</em>
```

Comment:
Are all these breaks necessary? Especially the first one between platform and adoption doesn't look right. Best check in preview also in different viewports (mobile vs desktop). Usually keeping away from artificial breaks works better to make the view dynamic.

Reply:
You caught my terrible UI dev skillz 😅 But seriously I have removed the breaks. I think that the issue I have is that the text is IMO too wide but I think it is still better to be more flexible with the layout.